h5py/api_functions.txt

```diff
@@ -22,6 +22,8 @@
 #
 # Function lines are C function signatures, optionally preceded by:
 #   MPI     If present, function requires an MPI-aware build of h5py
+#   ROS3    If present, function requires a ROS3 enabled build of h5py
+#   DIRECT_VFD If present, function requires an DIRECT VFD enabled build of h5py
```

Comment:
I think these should all say 'build of HDF5'

Reply:
Thank you!